qiskit/quantum_info/operators/symplectic/pauli_utils.py

```diff
@@ -10,27 +10,24 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 """
-PauliTable utility functions.
+PauliList utility functions.
 """
 
-import numpy as np
-from .pauli_table import PauliTable
+from qiskit.quantum_info.operators.symplectic.pauli_list import PauliList
 
 
 def pauli_basis(num_qubits, weight=False):
-    """Return the ordered PauliTable for the n-qubit Pauli basis.
+    """Return the ordered PauliList for the n-qubit Pauli basis.
 
     Args:
         num_qubits (int): number of qubits
         weight (bool): if True optionally return the basis sorted by Pauli weight
                        rather than lexicographic order (Default: False)
 
     Returns:
-        PauliTable: the PauliTable for the basis
+        PauliList: the PauliList for the basis
     """
-    pauli_1q = PauliTable(
-        np.array([[False, False], [True, False], [True, True], [False, True]], dtype=bool)
-    )
+    pauli_1q = PauliList(["I", "X", "Y", "Z"])
```

Comment:
Does `PauliList` inherit from `PauliTable` or is this a breaking API change?

Reply:
Thanks. You are right. This is API breaking change. I'll make another name function to do this.